pkg/obfuscator/domain.go

```diff
@@ -48,6 +48,7 @@ func (d *domainObfuscator) replaceDomains(input string) string {
 				replacement = obfuscatedBaseDomain
 			}
 			output = strings.ReplaceAll(output, m[0], replacement)
+			d.ReplacementTracker.AddReplacement(baseDomain, baseDomain, obfuscatedBaseDomain)
```

Comment:
Generators no longer write to the tracker. Each new value must be reported explicitly.

Reply:
sounds good, do you have the Replacement struct anywhere? I'm not seeing it in the diff